content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>And this also needs <code>os.path.join</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Yup, done